Security/centurylink-cloud-guide-to-vormetric-dsm.md

```diff
@@ -268,11 +268,11 @@ Every time a host user or application attempts to access a GuardPoint file, the
 ----------|------------|--------|-----------|----------|-----------
 **1**|&#42;|&#42;|SQL_Server_Set|&#42;|Permit<br>Apply Key
 **2**|&#42;|Admin_Set|&#42;|Read|Permit<br>Audit
-**3**|&#42;|&#42;|&#42;|&#42;|Deny<br>Audit
+**3**|&#42;|&#42;|&#42;|&#42;|Deny<br>Audit**
```

Comment:
Agreed. I did that so the Markdown would cooperate in Atom. Otherwise, Bold continued on for a couple paragraphs.